python/essex-config/essex_config/sources/env_source.py

```diff
@@ -47,6 +47,12 @@ def __get_data(
             raise FileNotFoundError(msg)
         return {}
 
+    def get_data(self) -> dict[str, Any]:
+        """Get the data from the environment."""
+        if not self._file_path:
```

Comment:
Does this mean that if we use env vars set with `export` and don't use a dotenv file  it will not substitute variables?